src/index/IndexImpl.Text.cpp

```diff
@@ -2,17 +2,19 @@
 // Chair of Algorithms and Data Structures.
 // Author: Björn Buchhold (buchhold@informatik.uni-freiburg.de)
 
+#include "./IndexImpl.h"
+
```

Comment:
This File was simply renamed (from Index to IndexImpl). Nothing else should be changed (I don't understand the whole text business anyway)